### DISCUSSION TITLE
Lengte Observatiedoelen in Datastroom

### DISCUSSION THREAD

**Discussion by kad-bloemy** [kadaster-labs/sensrnet-home#271] [Q&A]
De getoonde lengte van het veld Observatiedoelen in Datastroom in het invulscherm voor Datastroom is zeer beperkt. Ik kan een lange tekst invullen, maar al typend schuift de tekst naar links. Is niet heel overzichtelijk. Het feit dat in Datastroom een veld Observatiedoel voorkomt en ook nog eens als apart iets op het scherm is nogal dubbelzinnig. Ik zie hier ook geen toelichting bij. Kunnen de twee niet verschillende namen krijgen?

**Comment by CelineJansen** [comment]
Persoonlijk vind ik het wel duidelijk zo, juist omdat het in de tool twee gescheiden schermen zijn, maar als andere gebruikers stoort kan het veld "Observatiedoelen" binnen de datastroom misschien hernoemd worden naar "Gekoppelde Observatiedoelen" in de UI?

**Comment by CelineJansen** [comment]
Voor wat betreft het formaat van het veld: als ik alle 3 de huidige doelen koppel ziet het er zo uit: 
![image](https://user-images.githubusercontent.com/79267472/146407150-2abf69e6-50a9-47bb-8129-0f75ea5b50fd.png)

Misschien afhankelijk van gebruik kijken of het vaak voor komt dat er meerdere doelen bij 1 datastroom horen, en afhankelijk daarvan bepalen om het veld groter te maken. Ik denk dat er vaak maar sprake zal zijn van 0 of 1 doelen. Maar ben het met Yolanda eens dat het mooier is wanneer een veld naar onder toe groter wordt mocht dat nodig zijn ipv een horizontale verschuiving.

**Comment by CelineJansen** [comment]
![image](https://user-images.githubusercontent.com/79267472/146585437-b0832b86-f102-4496-923f-40208229b1ed.png)
Dit is trouwens wel minder netjes...

**Comment by kad-floriw** [comment]
Ik heb een ticket aangemaakt voor het correct weergeven van observatie-doelen in de centrale viewer.

**Reply by marcvanandel** [reply]
https://github.com/kadaster-labs/sensrnet-central-viewer/issues/48 om precies te zijn 😉
